### PR TITLE
Add storage import docs, fixes #140

### LINKED ISSUE #140: Add new docs for command-line physical storage container import task
**Relevant documentation branch**

* 2.6

**Affected pages**

* https://www.accesstomemory.org/docs/latest/admin-manual/maintenance/cli-import-export/#import-csv-files-via-the-command-line
* ("see also" link will be added to https://www.accesstomemory.org/docs/latest/user-manual/add-edit-content/physical-storage/)
* We will also need to add the new CSV template to the [CSV templates]() page on the wiki

**Issue description**

In AtoM 2.6, a new command-line task has been added that supports the import of physical storage data using a CSV as input. New documentation will be required to cover this functionality.

### PR DESCRIPTION
**Pull request by fiver-watson**
This PR adds new documentation for the command-line physical storage import task added to AtoM 2.6.

### CODE REVIEW

File: admin-manual/maintenance/cli-import-export.rst [hunk 1]
```diff
@@ -1642,6 +1643,224 @@ are intentionally importing duplicate deaccession records, you can use the
 
 :ref:`Back to top <cli-import-export>`
 
+.. _csv-import-storage-cli:
+
+Import physical storage containers and locations
+------------------------------------------------
+
+This task will allow for the import of physical storage data, as well as updates 
+to existing physical storage containers, via an accompanying CSV file. Read more
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: this is perhaps overly pedantic, but "accompanying CSV file" sounds like the import is accompanied by a CSV file - I think you can simply remove the word "accompanying"

File: admin-manual/maintenance/cli-import-export.rst [hunk 2]
```diff
@@ -1642,6 +1643,224 @@ are intentionally importing duplicate deaccession records, you can use the
...
+.. code-block:: bash
+
+   php symfony csv:physicalobject-import /path/to/storage.csv
+
+The CSV filename and path (shown as the example ``/path/to/storage.csv``
+above) must be a valid path to a CSV file that is readable by the current
+user.
+
+The CSV file must use UTF-8 character encoding if it includes characters
+outside of the basic ASCII character set.  The CSV file formatting must use a
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst:1666]: remove double space between sentences

File: admin-manual/maintenance/cli-import-export.rst [hunk 3]
```diff
@@ -1642,6 +1643,224 @@ are intentionally importing duplicate deaccession records, you can use the
...
+* Skip any unmatched rows
+* Update all matching records where multiple matches on container name are found
+* Allow for partial name matches to be considered acceptable matching criteria
+* Skip the first 10 data rows of the CSV
+* Report any non-fatal errors to a file called "errors.log"
+* Update the search index as the import progresses
+
+.. code-block:: bash
+
+   php synfony csv:physicalobject-import --update --skip-unmatched --multi-match="all" --partial-match --skip-rows=10 --error-log="/usr/share/nginx/atom/errors.log" --index /path/to/my/storage.csv
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: symfony

File: admin-manual/maintenance/cli-import-export.rst [hunk 4]
```diff
@@ -1642,6 +1643,224 @@ are intentionally importing duplicate deaccession records, you can use the
...
+
+This example could also be written as follows, use the short names for the options:
+
+.. code-block:: bash
+
+   php synfony csv:physicalobject-import -u -s --multi-match="all" -p -o=10 -l="/usr/share/nginx/atom/errors.log" -i /path/to/my/storage.csv
```
Comment (corinnemrogers) [inline admin-manual/maintenance/cli-import-export.rst]: symfony

**Review by corinnemrogers — CHANGES_REQUESTED** [review]
just a few minor edits, 2 discretionary and 2 typos

**Comment by fiver-watson** [comment]
Thanks for the review! I've implemented your changes, and squashed. Let me know if you see anything else, or if I should merge this now.

File: admin-manual/maintenance/cli-import-export.rst (hunk 2 above)
Comment (fiver-watson) [reply admin-manual/maintenance/cli-import-export.rst:1666]: Done

**Review by corinnemrogers — APPROVED** [review]
Looks good!

**Pull request merged**